public/views/index.html

```diff
@@ -2,7 +2,7 @@
 		<!-- Sidebar -->
 		<div id="sidebar-container" class="sidebar">
 			<div id="logout" class="sidebar" ng-model="user">
-				<a href="/">logout</a> | {{user.email}}
+				<a href="/" ng-click="logout()">logout</a> | {{user.email}}
```

Comment:
option 2 is better, but i think we should get rid of all this redirection. how about we just get rid of the `href="/"` part and leave everything as is? so it looks like a link and you think to click it but actually it's just an indirect link ;-)